flake.nix

```diff
@@ -140,7 +140,7 @@
                 };
 
                 remacsBindings = prev.rustPlatform.fetchCargoTarball {
-                  src = emacsNgSource + "/rust_src/remacs-bindings";
+                  src = emacsNgSource + "/rust_src/ng-bindgen";
```

Comment:
Do not change it. the Nix of emacs-ng on locked rev version.

Reply:
Thanks.